src/lib/components/unclaimed-project-card/unclaimed-project-card.svelte

```diff
@@ -53,13 +53,15 @@
 
   $: hasClaimableFunds = mergedUnclaimedFunds.length > 0;
 
-  $: unclaimedTokenPile = mergedUnclaimedFunds?.map((fund) => ({
-    component: Token,
-    props: {
-      address: fund.tokenAddress,
-      show: 'none',
-    },
-  }));
+  $: unclaimedTokenPile = mergedUnclaimedFunds
+    ?.map((fund) => ({
+      component: Token,
+      props: {
+        address: fund.tokenAddress,
+        show: 'none',
+      },
+    }))
+    .slice(0);
```

Comment:
what's the point of `slice(0)` here? 🤔

Reply:
oops, leftover from some previous ideas. removing